ofono/src/cell-info-dbus.c

```diff
@@ -104,6 +106,21 @@ static const struct cell_property cell_lte_properties [] = {
 	CELL_LTE_PROPERTY(0x800,timingAdvance)
 };
 
+static const struct cell_property cell_nr_properties [] = {
+	CELL_NR_PROPERTY(0x001,mcc),
+	CELL_NR_PROPERTY(0x002,mnc),
+	CELL_NR_PROPERTY(0x004,nci),
```

Comment:
Doesn't `nci` need to be special-cased, since it's 64-bit?

Reply:
Added support for 64-bit property values.